akka-stream-tck/src/test/scala/akka/stream/tck/FusableProcessorTest.scala

```diff
@@ -18,7 +18,7 @@ class FusableProcessorTest extends AkkaIdentityProcessorVerification[Int] {
     implicit val materializer = ActorMaterializer(settings)(system)
 
     // withAttributes "wraps" the underlying identity and protects it from automatic removal
-    Flow[Int].andThen[Int](Identity()).named("identity").toProcessor.run()
+    Flow[Int].via(Stages.identityGraph.asInstanceOf[Graph[FlowShape[Int, Int], Unit]]).named("identity").toProcessor.run()
```

Comment:
Yes, you remember correctly. I think I managed to get the solution quite slick, if I may say so :)